configs/jest.config.js

```diff
@@ -1,5 +1,5 @@
+/** @type {import('ts-jest').JestConfigWithTsJest} */
 module.exports = {
     rootDir: '../',
-    projects: ['<rootDir>/applications/*', '<rootDir>/extensions/*', '<rootDir>/packages/*'],
-    reporters: ['default', ['jest-junit', { outputDirectory: 'unit-test-results', outputName: 'jest-report.xml' }]]
```

Comment:
This change is leading to the Jest extension in VSCode not to work anymore. I am trying to fix this, but I guess Jest needs to run either for ModuleJS or ESM?

Reply:
The Jest extension in VSCode is using the "jest.jestCommandLine" setting to execute tests (https://github.com/CrossBreezeNL/crossmodel/blob/b94137d75b41879c858d32a9e5c5b75ffaaf7a92/.vscode/settings.json#L43C3-L43C77).

I tried to change it to the setting for ESM for testing the extension (since this is the only package which has unit tests a.t.m. I tried (amonst other things):

"jest.jestCommandLine": "npm exec -- cross-env NODE_OPTIONS=--experimental-vm-modules NODE_NO_WARNINGS=1 jest --config=./configs/esm.jest.config.js"

But while this works fine on the command line (also with --watch and --watchAll) it doesn't work with the extension.

Do you have any idea?

Reply:
I think without changing anything the tests work in all other projects (except the extension). But once I change the command to execute the extension jest stuff it doesn't work.

Maybe for now it's acceptable, I just need to remember to run the extension tests seperately.

Reply:
I adapted everything a bit and the extension works better now. However I didn't get the debugging support to work properly in the Test view.

Reply:
Debugging I can do with de VS Code launch. I like the extension because it run the test automatically when you change a file.

It works now also within my setup, great!